File: C#/Estrutura sequencial/medidas.cs
// Variáveis
double a, b, c, areaQuadrado, areaTriangulo, areaTrapezio;

// Entrada de dados
Console.Write("Digite a medida A: ");
a = double.Parse(Console.ReadLine());
Console.Write("Digite a medida B: ");
b = double.Parse(Console.ReadLine());
Console.Write("Digite a medida C: ");
c = double.Parse(Console.ReadLine());

// Algoritmo matemático
areaQuadrado = Math.Pow(a, 2);
areaTriangulo = (a * b) / 2;
areaTrapezio = (a + b) * c / 2;

// Saída de dados
Console.Write("A área do quadrado é " + areaQuadrado.ToString("F4") + ", do triângulo " + areaTriangulo.ToString("F4") + " e do trapézio é " + areaTrapezio.ToString("F4"));

File: Java/Estrutura sequencial/medidas.java
// Importando biblioteca
import java.util.Scanner;

public class medidas {

	public static void main(String[] args) {
		
		// Variáveis
		double a, b, c, areaQuadrado, areaTriangulo, areaTrapezio;

		Scanner sc = new Scanner(System.in); // Para facilitar a leitura de dados
		
		// Entrada de dados
		System.out.print("Digite a medida A: ");
		a = sc.nextDouble();
		System.out.print("Digite a medida B: ");
		b = sc.nextDouble();
		System.out.print("Digite a medida B: ");
		c = sc.nextDouble();
		
		// Algoritmo matemático
		areaQuadrado = Math.pow(a, 2);
		areaTriangulo = (a*b)/2;
		areaTrapezio = (a+b)*c/2;
		
		// Saída de dados
		System.out.print("A área do quadrado é " + String.format("%.4f", areaQuadrado) +  ", do triângulo é " + String.format("%.4f", areaTriangulo) + " e do trapézio é " + String.format("%.4f", areaTrapezio));
		
		sc.close(); // Boa prática de programação
	}
}